repos.json

```diff
@@ -260,6 +260,7 @@
     { "language": "python", "repo": "googleapis/python-cloudbuild", "apiHint": "cloudbuild"},
     { "language": "python", "repo": "googleapis/python-container", "apiHint": "container"},
     { "language": "python", "repo": "googleapis/python-containeranalysis", "apiHint": "containeranalysis"},
+    { "language": "python", "repo": "googleapis/python-crc32c", "apiHint": "crc32c"},
```

Comment:
Is `crc32c` a real Google Cloud API?  We generally use this for tracking the shortname that should be used for team association purposes.  Is this only used for storage, or is it a general use lib?

Reply:
general use.  but needs to be tracked for releasetool